### PR TITLE
Fix yorkie version 0.1.0

### PR DESCRIPTION
**Pull request by ppeeou**
#### What does this PR do?

I think codepair is a service, so it looks good to fix the version

### Checklist

- [ ] Added relevant tests
- [x] Didn't break anything

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ services:
   yorkie:
     volumes:
     - ./yorkie.json:/app/yorkie.json
-    image: 'yorkieteam/yorkie:latest'
+    image: 'yorkieteam/yorkie:0.1.0'
```
Comment (hackerwins) [inline docker/docker-compose.yml:19]: We recently released a new version. How about using v0.1.1?

- https://hub.docker.com/repository/docker/yorkieteam/yorkie/tags
- https://www.npmjs.com/package/yorkie-js-sdk

**Review by hackerwins — COMMENTED** [review]
Thank you. Leave a brief comment.

File: docker/docker-compose.yml (hunk 1 above)
Comment (ppeeou) [reply docker/docker-compose.yml:19]: @hackerwins There seem to be a bug in using 0.1.1 

The cause has not been determined yet, so I fixed it at 0.1.0

Or Would I find the cause and create a PR that fixes 0.1.1?

Comment (hackerwins) [reply docker/docker-compose.yml:19]: I didn't know there was a bug in v0.1.1. 🤔 First use v0.1.0 and would you like to report the bug?

**Pull request merged**